mkt/developers/tasks.py

```diff
@@ -61,8 +62,18 @@ def validator(upload_id, **kw):
         log.info(u'[FileUpload:%s] Does not exist.' % upload_id)
         return
 
+    temp_path = None
+    # Make a copy of the file if it's a packaged app since we can't assume the
+    # uploaded file is on the local filesystem.
+    if upload.name.endswith('.zip'):
```

Comment:
Do we have something more reliable than that ? Or are we already doing that everywhere anyway ? It seems fragile. I'd rather do something like `if not upload.name.startswith(('https://', 'http://'))`